hyperspy/drawing/_markers/point.py

```diff
@@ -87,7 +87,7 @@ def update(self):
                                  self.get_data_position('y1')])
         self.marker._sizes = [self.get_data_position('size')]
 
-    def plot(self):
+    def plot(self, fig_draw_animated=True):
```

Comment:
https://github.com/hyperspy/hyperspy/pull/1431/commits/99f296dae363f226ba844e6bf613822fceb0d74d really reduced the duration when closing a figure with many markers. With a figure with 500 markers, it went from 3 minutes to 1 second.

Similar improvement when opening a plot with 500 markers.

I'll rename the argument, and implement them for the other markers.